content/pt/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Projeto OpenTelemetry
+description: >-
+ Telemetria de alta qualidade, abrangente e portátil para permitir uma observabilidade eficaz
+developer_note:
+  O shortcode blocks/cover (usado abaixo) vai servir como imagem de background
+  para qualquer arquivo de imagem que contenha "background" no nome.
+default_lang_commit: e89a198
```

Comment:
This commit is quite old. Please review this page carefully to ensure that it corresponds to the `en` page (you'll want to pay particular attention to link references that have changes from absolute to relative).

Reply:
Thanks! I'll take a look right away

Reply:
@chalin,

 I've added the commit based on [these](https://opentelemetry.io/docs/contributing/localization/#adding-default_lang_commit-to-new-pages) docs and this output from the `scripts/check-i18n.sh -i`:
```bash
08799298a05d17a6a659821bfe1c9d994e4ceba9 - hash of 'main' at HEAD
```

Regarding the content, I reviewed it and it is up to date, but I didn't find any change related to the links (at `_index.md`), could you please share an example that changed?

Reply:
Did you run that command to see what the detailed diff was for the index page? That would have helped you see what changed.

In particular, the links into `/docs` should all be page relative (that is, they should not start with a slash).

That is, this PR contains:

```markdown
- [Saiba mais](/docs/what-is-opentelemetry/)
- [Experimente a demonstração](/docs/demo/)
```

But the [main English page](https://github.com/open-telemetry/opentelemetry.io/blame/main/content/en/_index.md) now has the following link (notice that there is no leading slash):

```markdown
- [Learn more](docs/what-is-opentelemetry/)
- [Try the demo](docs/demo/)
```

Carefully review the rest of the file too.

Reply:
Thank you, done!! My fault :sweat_smile: